### PR TITLE
Update to java15

### PR DESCRIPTION
**Pull request by dragutin-nav**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -15,15 +15,23 @@
...
         <confluent.version>5.5.1</confluent.version>
         <avro.version>1.10.2</avro.version>
         <veilarbregistrering.skjema.version>1.202108021530-9a956c3</veilarbregistrering.skjema.version>
         <jaxb.version>2.3.1</jaxb.version>
+
         <testcontainers.version>1.16.0</testcontainers.version>
+
+        <sonar.projectKey>navikt_veilarbportefolje</sonar.projectKey>
+        <sonar.organization>navit</sonar.organization>
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
```
Comment (stianStensli) [inline pom.xml:31]: I removed sonar in dev

File: pom.xml [hunk 2]
```diff
@@ -274,6 +282,11 @@
             <artifactId>shedlock-provider-jdbc-template</artifactId>
             <version>4.28.0</version>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.11.0</version>
+        </dependency>
```
[the remark this answers is not in the document]
Comment (dragutin-nav) [reply pom.xml]: we will find out soon, its removed

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -56,12 +56,12 @@ jobs:
     if: github.ref == 'refs/heads/dev' || github.ref == 'refs/heads/master'
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
-      - name: Set up JDK 11
+        uses: actions/checkout@v1
```
Comment (stianStensli) [inline .github/workflows/main.yml:59]: Can we use checkout@v2 ?

Comment (dragutin-nav) [reply .github/workflows/main.yml]: Done

**Review by stianStensli — APPROVED** [review]

**Pull request merged**